Enterprise-CMCS/eAPD
# Tbolt/4523 Dont allow blank strings in text input fields

Resolves #4523 

### Description
This updates all of the validations and related code to make empty strings not pass the validation

### Automated test cases written
Related tests updated to work with the new change

### Steps to manually verify this change

1. Create both types of APDs
2. Verify that for all text input fields (including rich text) you can enter blank spaces but it won't pass the validation or admin check

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
